src/pages/home/report/ReportActionsView.js

```diff
@@ -174,53 +176,79 @@ class ReportActionsView extends React.Component {
     }
 
     componentDidUpdate(prevProps) {
+        const isReportFullyVisible = this.getIsReportFullyVisible();
+
+        // NETWORK CHANGE DETECTED
```

Comment:
Not a fan of these capitalized comments, can we make them lowercase and keep consistency with all other comments, or use variable to communicate this? Eg,
```js
const bool wasNetworkChangeDetected = lodashGet(prevProps.network, 'isOffline') && !lodashGet(this.props.network, 'isOffline');
...
if (wasNetworkChangeDetected) {
...
```

Reply:
With the comments I wanted to create a sense that these are discrete "events" or "situations" we might encounter. 

I had a draft where each of these was moved to a method like `checkIfNetworkChangedAndRefetch()`, but that felt like something Uncle Bob would do and maybe overkill. I eventually just decided to do something obvious and ugly. Will try the variable thing! Thanks!

Reply:
Making these comments not be all uppercase already solves my concern. I think we should follow our guidelines, so others can as well. If we break consistency for our comments, contributors may start doing the same

Reply:
And to be clear, I did appreciate the comments that are in this code, as it would've been much harder to follow without them